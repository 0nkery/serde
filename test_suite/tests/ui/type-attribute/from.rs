#[macro_use]
extern crate serde_derive;

#[derive(Deserialize)]
#[serde(from = "Option<T")]
enum TestOne {
    Testing,
    One,
    Two,
    Three,
}

fn main() {}
