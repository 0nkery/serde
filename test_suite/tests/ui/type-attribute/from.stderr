error: failed to parse type: from = "Option<T"
 --> $DIR/from.rs:4:10
  |
4 | #[derive(Deserialize)]
  |          ^^^^^^^^^^^

error: aborting due to previous error

